test/reactionsystem_core/events.jl

```diff
@@ -313,6 +309,66 @@ end
 
 ### Additional Correctness Tests ###
 
+# Tests that events are properly triggered for SDEs.
```

Comment:
Makes stochastic models where all types of events should trigger. Trigger updates a parameter, which is used to check that they were triggered.